crates/red_knot_test/src/parser.rs

```diff
@@ -1801,4 +1808,41 @@ mod tests {
              (including embedded files).",
         );
     }
+
+    #[test]
+    fn obvious_typos_in_directives_are_detected() {
+        let source = dedent(
+            "
+            # Some header
+            <!-- snpshotttt-digggggnosstic -->
```

Comment:
Fair warning, I'm going to ask you to pronounce this at our next standup.

Reply:
hahaha, I had to think up an _outrageous_ typo or the `typos` pre-commmit check kept on complaining that there was a typo in the test here